### ISSUE TITLE
Tool status

### ISSUE THREAD

**Issue by runcom** [containers/oci-fetch#4]
/cc @philips @sgotti @dgonyeo @mrunalp @rhatdan

Hey, following my discussion at https://github.com/containers/oci-fetch/pull/2, I'd like to understand if and how this tool is needed. Right now, what this tool can do is already present in both `containers/image` and `projectatomic/skopeo` and I found myself reviewing #2 when `skopeo` and `image` already had that functionality.
This seems a waste of effort to me. Can we concentrate ourselves in `skopeo` and `containers`? `skopeo` already does what `oci-fetch` does with `skopeo copy docker://fedora oci:fedora-oci-image-layout` also. And more important this tool is duplicating the effort in `containers/image` because there it lives the library to interact with registries and containers' images.

**Comment by philips** [comment]
@runcom as discussed on the phone last Friday I think we should have the oci-fetch tool as a relaly simple wrapper around the github.com/containers/image library.  We can move oci-fetch into the image library but there is value in a "pure" OCI only fetching tool to illustrate the use case and help people test their registries.

**Comment by runcom** [comment]
@philips yes, I'd love to see this tool refactored to use the already existing API exposed in containers/image to download OCI image layout to avoid duplicating effort.

**Comment by philips** [comment]
@runcom @dgonyeo has this on his personal TODO list now.
